examples/with-thirdpartyemailpassword-2fa-passwordless/src/SecondFactor/index.tsx

```diff
@@ -30,7 +30,7 @@ export default function SecondFactor() {
                 alignItems: "center",
                 justifyContent: "center",
             }}>
-            <Passwordless.SignInUp />
+            <Passwordless.SignInUp redirectOnSessionExists={false} />
```

Comment:
this hasn’t made the example apps simpler.. so what is the benefit of doing this? Also, it has introduced a bug in which if you are logged in, and you visit the second factor screen, it keeps you on that screen (it sometimes navigates you away, but it glitches).